File: back/back/Models/Sony.cs
namespace back.Models
{
    public class Sony
    {
        public string Notes { get; set; }
        public string SerialNumber { get; set; }
        public string Type { get; set; }
        public decimal Price { get; set; }
        public Sony(string Serial, string Ty, decimal Pr, string Notes)
        {
            this.Notes = Notes;
            SerialNumber = Serial;
            Type = Ty;
            Price = Pr;
        }
        public Sony() { }
    }
}

File: front/src/app/models/Sony.ts
export class Sony {
    serialNumber:string;
    notes:string;
    type:string;
    price:number;

    constructor(ser:string, type:string, notes:string, price:number) 
    {
        this.serialNumber = ser;
        this.type = type;
        this.notes = notes;
        this.price = price;
    }
}